variables.tf

```diff
@@ -66,6 +66,18 @@ variable "subnet_ids" {
   type        = list(string)
 }
 
+variable "node_role_arn" {
+  type        = list(string)
+  default     = []
+  description = "If provided, assign workers the given role, which this module will not modify"
+  validation {
+    condition = (
+      length(var.node_role_arn) < 2
+    )
+    error_message = "You may not specify more than one `node_role_arn`."
```

Comment:
why is it a list if it can't be more than one element?

Reply:
This is a relatively well-known hack in the Terraform world. It is required because of this [`join`](https://github.com/35up/terraform-aws-eks-node-group/pull/1/files#diff-dc46acf24afd63ef8c556b77c126ccc6e578bc87e3aa09a931f33d9bf2532fbbR60).